pkg/httpmetrics/github_api.go

```diff
@@ -155,14 +210,6 @@ func bucketizePath(_ context.Context, path string) string {
 
 func instrumentGitHubAPI(next http.RoundTripper) promhttp.RoundTripperFunc {
```

Comment:
This does... nothing?

Reply:
I think this looks good, if you remove this and it's caller.

Reply:
Done